spec/decorators/playlist_decorator_spec.rb

```diff
@@ -22,14 +22,14 @@
   end
 
   describe "#title" do
-    it "uses the label" do
-      expect(decorator.title).to eq playlist.label
+    it "uses the title" do
+      expect(decorator.title).to eq playlist.title
     end
   end
 
   describe "#titles" do
-    it "delegates to the label of the Playlist" do
-      expect(decorator.label).to eq(["My Playlist"])
+    it "delegates to the title of the Playlist" do
+      expect(decorator.titles).to eq(["My Playlist"])
```

Comment:
is this title?

Reply:
Yes, it was necessary for covering: https://github.com/pulibrary/figgy/pull/2163/files#diff-35173295a238f26d381e957b87912e6dR22.  I can reword this test description.